### DISCUSSION TITLE
Communicating with other stations, monsters interacting with the radar, the disease "system".

### DISCUSSION THREAD

**Discussion by Sol0D** [FakeFishGames/Barotrauma#7471] [Ideas and suggestions]
when you complete a mission of the type kill a monster, you have to use a communicator (via radio, satellite) to alert the station that you have killed him, and try to exploit more that mechanics of being able to communicate with stations or other submarines something like when you're going back to the station to complete the mission pirates can intercept the signal and go to your location to steal your loot, and also implement a creature that is guided by the waves of the communicator or radar and that somehow the creature can interact with the radar waves, it could be a blind monster and if you turn off the radar the monster leaves or some monster that calls a bigger one if it is alerted by the radar making you have to sneak past it, I would also like to see some kind of disease system similar to darkest dungeon maybe not the best example but you get a certain disease your stats decrease, I am not a game developer and maybe what I am saying is a lot of bullshit but something like this I would like to see, if any of the developers get to read this lov ur game <3

**Comment by MateusJK15** [comment]
Radar, radio and satellite communication, all these 3 things DOES NOT WORK underwater.  Radar doesn't even exist in the game. Electromagnetic waves propagates poorly underwater, so even a powerful antenna wouldn't be able to send a signal far away, that's why submarines never use radio underwater.

**Reply by Sol0D** [reply]
when I say "radar" I mean the sonar of the navigation terminal and when the submarine launches the sonar ping a monster can pick up that signal or sound, it makes sense that radio waves or satellites do not work underwater.

**Reply by MateusJK15** [reply]
This has been in the game for decades, molochs are blind and ignore your sub if you're not moving and have the reactor turned off, but if you use an active ping near them they get REALLY pissed off and rush to kill you. Every monster in the game reacts to active pings.

**Comment by MaaiG** [comment]
That’s not entirely true. Anyway, we have a game with a submarine in space. So you can talk about anything. ))
At least, because no one is ashamed that the [air does not come out](https://github.com/Regalis11/Barotrauma/issues/7466) of the open airlock! ))

**Reply by Sol0D** [reply]
the other things may not make as much sense but it would be cool to see a blind creature that can call out to a larger creature or horde by scaring it or making a sound and having to sneak past it.

**Reply by MateusJK15** [reply]
You mean a creature that can call for reinforcements when attacked ? That's already in the game.

**Reply by Sol0D** [reply]
Okay, almost everything I say is already in the game, got it.
